ui/mirage/factories/alloc-file.js

```diff
@@ -54,7 +54,7 @@ const fileBodyMapping = {
 export default Factory.extend({
   id: i => i,
 
-  isDir: faker.random.boolean,
+  isDir: faker.random.boolean(),
```

Comment:
Should this be `isDir: () => faker.random.boolean()` like the others below?

Also, why doesn't passing the function as the value work? Binding issue?

Reply:
You’re right!

And good question about passing the function, I entirely missed that aspect. It’s  unnecessary to wrap in an inline function when the `boolean` function can be passed directly. So thanks for asking, I’ve reverted those changes.